### ISSUE TITLE
add pictures for heroes and abilities

### ISSUE THREAD

**Issue by smidaharoun** [jamesmcfadden/overwatch-api#39]
I wonder if you're planning to add pictures for heroes and abilities.
In case if you are planning, I suggest you check [ this repository](https://github.com/Resike/Overwatch), it contains almost all resources of Overwatch

**Comment by jamesmcfadden** [comment]
Hi @smidaharoun. I think this would be a nice addition, at least the portraits of each hero. At the moment I have no spare time to add this but will look at doing so in the future.

**Comment by smidaharoun** [comment]
Yeah it's would be great, I would like to help you but unfortunately I have not that skilled in php.
Just another suggestion, I would be better if you add the hero's role field in `getHeroes` WS

**Comment by MTechDE** [comment]
Any update?
